config/samples/nnf_v1alpha1_nnfcontainerprofile.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: nnf.cray.hpe.com/v1alpha1
```

Comment:
Delete or move to ./config/examples if you plan to use it

Reply:
My thought was to keep this as a fully documented sample with all of the options that could be used as a reference. I don't plan on this one getting deployed like the ones in the examples folder.

Reply:
In that case can you make the spec.containers.command less insane

Reply:
I'll make it just return 0 after 15 seconds.